.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
 node_js:
   - "0.10"
+  - "0.12"
 script: make travis
```

Comment:
Well, I agree, but last time you said you wanted to keep the makefile. Personally, I'm in favor of make.js since it works everywhere.